gui/src/app/FileEditor/TextEditor.tsx

```diff
@@ -227,4 +242,24 @@ const NotSelectable: FunctionComponent<PropsWithChildren> = ({ children }) => {
   return <div className="NotSelectable">{children}</div>;
 };
 
+const createHintTextContentWidget = (hintText: string) => {
+  return {
+    getDomNode: () => {
+      const node = document.createElement("div");
+      node.style.width = "max-content";
+      node.style.pointerEvents = "none";
+      node.textContent = hintText;
+      node.style.fontStyle = "italic";
```

Comment:
Thoughts on pulling these `.style`s into a class? In particular, it would also be nice to make the text slightly lower contrast than it is currently, as vscode does

Reply:
I have created a css class EditorHintText for the italics and color. I feel like pointerEvents still belongs inline though.